gstreamer-gl/src/gl_context.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2018 Víctor Jáquez <vjaquez@igalia.com>
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use ffi;
+use glib::translate::*;
+use libc::uintptr_t;
+use GLContext;
+use GLDisplay;
+use GLPlatform;
+use GLAPI;
+
+impl GLContext {
+    pub fn new_wrapped(
+        display: &GLDisplay,
+        handle: uintptr_t,
```

Comment:
This function must be unsafe. If you pass a wrong handle in there, things will crash or worse :)

Reply:
aha!

Right now I'm cooking an example with this code. I'm aiming to use a  simple window with [glutin](https://crates.io/crates/glutin) and sharing the context to a pipeline of type ``videotestsrc ! glupload ! glcolorconvert ! appsink``. Thus, I'll avoid the guessing :)

Reply:
That example sounds very useful indeed, feel free to put it under the examples subcrate here :)